crates/ironrdp-web/src/session.rs

```diff
@@ -50,6 +50,8 @@ struct SessionBuilderInner {
     hide_pointer_callback_context: Option<JsValue>,
     show_pointer_callback: Option<js_sys::Function>,
     show_pointer_callback_context: Option<JsValue>,
+
+    kdc_proxy_url: Option<String>,
```

Comment:
No need to separate this from the rest, can go after `pcb`

Reply:
updated